week2-section2/exercise2/Spotify.cpp

```diff
@@ -0,0 +1,181 @@
+#include "Spotify.h"
+#include "PlaylistContainer.h"
+#include <string>
+
+FSpotify::FSpotify()
+{
+	ActiveMenuCommand = EMenuCommand::MainMenu;
+	PlaylistContainer = FPlaylistContainer(0);
+};
+
+//------------------------------------------------------
+void FSpotify::OpenMainMenu() 
+{
+	std::system("cls");
+	std::cout << "Welcome to Spotify! What would you like to do?\n1 - Create a Playlist\n2 - Browse Playlists\n3 - Exit" << std::endl;
+	int InputConverter;
```

Comment:
This isn't really an `InputConverter` it's just the user's input. The name should describe what the variable _is_, not what it is possibly _used for_. So `UserInput` should be more than enough :)